torch/onnx/symbolic.py

```diff
@@ -421,6 +421,29 @@ def upsample_nearest2d(g, input, scale_factor):
                 height_scale_f=scale_factor, mode_s="nearest")
 
 
+def upsample_bilinear2d(g, input, output_size):
+    w_scale = float(output_size[-1]) / input.type().sizes()[-1]
+    h_scale = float(output_size[-2]) / input.type().sizes()[-2]
+    return g.op("Upsample", input, width_scale_f=w_scale,
+                height_scale_f=h_scale, mode_s="bilinear")
+
+
+def stack(g, *tensors, dim):
+    return g.op("ATen", *tensors, operator_s="stack", dim_i=dim)
+
+
+def nonzero(g, input):
+    return g.op("ATen", input, operator_s="nonzero")
+
+
+def round(g, input):
+    return g.op("ATen", input, operator_s="round")
+
+
+def gt(g, input, other):
```

Comment:
I believe you can directly emit the ONNX `Greater` op here rather than using ATen: https://github.com/onnx/onnx/blob/master/docs/Operators.md#greater

Reply:
Sure! I would probably submit that before the end of the day.

Reply:
Sure! I would submit that before the end of the day.